cosmos/lib/cosmos/microservices/reducer_microservice.rb

```diff
@@ -250,11 +247,12 @@ def process_file(filename, type, entry_seconds, file_seconds)
       )
       true
     rescue => e
-      if File.exist(filename)
-        Logger.error("Reducer Error: #{filename}:#{File.size(filename)} bytes: \n#{e.formatted}")
+      if file.local_path and File.exist?(file.local_path)
+        Logger.error("Reducer Error: #{filename}:#{File.size(file.local_path)} bytes: \n#{e.formatted}")
       else
-        Logger.error("Reducer Error: #{filename}:(Does Not Exist): \n#{e.formatted}")
+        Logger.error("Reducer Error: #{filename}:(Not Retrieved): \n#{e.formatted}")
       end
+      false
```

Comment:
We're not really using the return value (true/false) anymore.  ReducerModel.rm_file(file) everytime. Not sure why that was in an if before but I don't think we ever returned false.